src/DeprecatedHelpers/MyCollection/AddCollectedArtwork.ts

```diff
@@ -10,10 +13,13 @@ import { AddCollectedArtwork } from "../../Schema/Events/MyCollection"
  * ```
  *
  */
-export const addCollectedArtwork = (): AddCollectedArtwork => {
+export const addCollectedArtwork = ({
+  platform,
+}: AddCollectedArtworkArgs): AddCollectedArtwork => {
   return {
     action: ActionType.addCollectedArtwork,
     context_module: ContextModule.myCollectionHome,
     context_owner_type: OwnerType.myCollection,
+    platform,
```

Comment:
is updating this helper going to break the current implementation that we have for tracking?

Reply:
yes - we will have to update Eigen as well

Reply:
no need to update this though since we only want to interfaces from cohesion but not actions